src/kbmod/wcs_utils.py

```diff
@@ -195,7 +195,7 @@ def calc_actual_image_fov(wcs):
     Notes
     -----
     While this will work for any WCS, it is intended to be used with
-    tangent plane projections where the reference pixel is on the image.
+    tangent plane projections where the reference pixrrylyel is on the image.
```

Comment:
Is this a typo? I'm not sure what "pixrrylyel" means.

Reply:
yes my cat must've jumped on my keyboard 😂